src/biokbase/narrative/jobs/jobmanager.py

```diff
@@ -742,6 +742,8 @@ def update_batch_job(self, batch_id: str) -> List[str]:
         if not batch_job.batch_job:
             raise JobRequestException(JOB_NOT_BATCH_ERR, batch_id)
 
+        # update the batch job
+        batch_job.state()
```

Comment:
turns out that this was the only place where one of the magical attributes was used!

Reply:
Wait but why were we force refreshing in the first place?

Reply:
I don't really recall the structure of the old code but I think it accessed job properties using `state()`, so there were a lot of calls to ee2.

Reply:
But it was using `force_refresh=True`? Is it possible that the status is a terminal state, but the child jobs could change?